browser/lib/consts.js

```diff
@@ -1,10 +1,9 @@
 const path = require('path')
 const fs = require('sander')
 const { remote } = require('electron')
-const { app } = remote
```

Comment:
I couldn't make `app` extracted from `remote` while running tests, following error throws.
```bash
var app = remote.app;
                ^

TypeError: Cannot read property 'app' of undefined
```
Any ideas why this happened?

Reply:
I assume we have to make `remote` Mockable.

Reply:
As result, `require('electron')` just return the path of electron binary, a string.

I think it's an good idea to mock `electron`, so I created a new mock helper in `tests/helpers/setup-electron-mock.js`, we can add more mocked function into it if needed.